capstone/src/main/java/com/step902020/capstone/EventTemp.java

```diff
@@ -3,24 +3,43 @@
 import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
 import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
 import org.springframework.data.annotation.Id;
+import java.util.*;
 
 @Entity(name = "eventTemp")
 public class EventTemp {
 
   @Id
   Long datastoreId;
 
-  String text;
+  String name;
 
-  public EventTemp(String text) {
-    this.text = text;
+  List<Review> reviews;
```

Comment:
Just out of curiosity because I am facing the same issue: will you eventually be having a separate "Review" entity and instead store ids of these reviews, or will this list be storing actual Review objects?

Reply:
Oh I just stored the Review obj  in the Event's list